workflows/src/main/kotlin/com/r3/corda/lib/accounts/workflows/flows/RequestKeyForAccountFlows.kt

```diff
@@ -54,9 +53,6 @@ class RequestKeyForAccountFlow(
                     // The PublicKeyHashToAccountIdMapping table has a primary key constraint over PublicKey, therefore
                     // a key can only ever be stored once. If you try to store a key twice, then an exception will be
                     // thrown in respect of the primary key constraint violation.
-                    serviceHub.withEntityManager {
```

Comment:
This is because we now do the registering on both nodes in the CI flows right?

Reply:
Yeah that's right